homework/02LessonTwo.js

```diff
@@ -59,7 +59,8 @@ function add(number1, number2) {
 // this function will add three numbers
 // you must call your function add() in addThree()
  function addThree (number1, number2, number3) {
-   return number1 + number2 + number3;
+   var x= add(number1, number2)
+   return add(x, number3) 
```

Comment:
Yes I had that but it says that we need to use the add function so I changed it to have that.      

Sent from my iPhone

> On Aug 8, 2016, at 1:45 PM, Piero A. Pretto <notifications@github.com> wrote:
> 
> In homework/02LessonTwo.js:
> 
> > @@ -59,7 +59,8 @@ function add(number1, number2) {
> >  // this function will add three numbers
> >  // you must call your function add() in addThree()
> >   function addThree (number1, number2, number3) {
> > -   return number1 + number2 + number3;
> > +   var x= add(number1, number2)
> > +   return add(x, number3) 
> You also write:
> 
> "return number1 + number2 + number3" instead of using an extra variable, "x", to store your logic.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>